### ISSUE TITLE
Make storage of annotation property values more predictable

### ISSUE THREAD

**Issue by ledsoft** [kbss-cvut/jopa#175]
Currently, string annotation property values are stored based on their content - if the string is determined to be a valid absolute IRI/URI, it is stored as an individual IRI. This can lead to unexpected and hard to explain/debug issues (#167).

Therefore, annotation property values handling should be reworked as follows:
- Only values of type `URI`/`URL`/`IRI` will be stored as OWL individuals.
- String values will be stored as strings with language tag configured for the persistence unit/attribute.
- Literal values of other types will be stored according to their type mapping.

Basically, unless the value is of type `URI`/`URL`/`IRI`, it is treated exactly the same way as if it were a datatype property value.
